### PR TITLE
Update the URL for our enhanced source view

### PR DESCRIPTION
**Pull request by jaredpar**

### CODE REVIEW

**Comment by dpoeschl** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ From the NuGet package manager console:
 ### Source code
 
 * Clone the sources: `git clone https://github.com/dotnet/roslyn.git`
-* [Enhanced source view](http://source.roslyn.codeplex.com/), powered by Roslyn 
+* [Enhanced source view](http://source.roslyn.io/), powered by Roslyn
```
Comment (jaredpar) [inline README.md]: Did not typo for once.

@jasonmalinowski shouldn't have left early

**Comment by agocke** [comment]
:shipit:

**Comment by jmarolf** [comment]
:+1:

**Comment by agocke** [comment]
Ship it

**Pull request merged**